assets/app/scripts/services/buildConfigs.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory('BuildConfigsService', function(DataService, $filter){
```

Comment:
I'd put this in the existing `BuildsService` rather than creating a new service for build configs. The builds service already does things with configs.